src/status_im/ui/components/react.cljs

```diff
@@ -80,14 +80,13 @@
 (defn text
   ([t]
    (reagent/as-element [text-class t]))
-  ([{:keys [uppercase?] :as opts} t & ts]
+  ([{:keys [uppercase? force-uppercase?] :as opts} t & ts]
    (reagent/as-element
-     (let [ts (cond->> (conj ts t)
-                       uppercase? (map #(when % (string/upper-case %))))]
-       (vec (concat
-              [text-class (add-font-style :style opts)]
-              ts))))))
-
+    (let [ts (cond->> (conj ts t)
+               (or force-uppercase? (and uppercase? platform/android?)) (map #(when % (string/upper-case %))))]
+      (vec (concat
+            [text-class (add-font-style :style opts)]
+            ts))))))
```

Comment:
I think that the readability is an issue here. And there are plenty of ways of improving it. For instance, you can easily move the `map` to a separate function, and replace all this `let` with something really simple, like this:
```
(->> (conj ts t)
     (transform-to-upper-case opts)
     (concat [text-class (add-font-style :style opts)])
     (vec))
```